src/ActorUtil.h

```diff
@@ -42,6 +42,19 @@ enum FileType
 	NUM_FileType,
 	FileType_Invalid
 };
+namespace std
+{
+	template<>
+		struct hash<FileType>
+	{
+		typedef FileType argument_type;
+		typedef std::size_t result_type;
+		result_type operator()(argument_type const& s) const
```

Comment:
Meh, I was just following the example.  I didn't want to delve into why they defined a result type.